ideascube/templatetags/ideascube_tags.py

```diff
@@ -184,3 +184,11 @@ def remove_qs(context, **kwargs):
     params.update(MultiValueDict(existing))
     params.pop('page', None)  # Changing search context, reset page.
     return '?%s' % params.urlencode()
+
+
+@register.simple_tag
+def media(instance, attribute):
```

Comment:
The `media` name seems wrong here.
`media` is for what use the tag, not what the tag is doing.
Without the commit comment or the media source code, it is difficult to know what this tag is doing.

We could rename it something like `gen_timed_url` or `unique_url` (I'm open to better idea)